tests/data_context/test_data_context.py

```diff
@@ -698,111 +698,6 @@ def test_data_context_is_project_initialized_returns_false_when_config_variable_
     assert FileDataContext.is_project_initialized(ge_dir) is False
 
 
-@pytest.mark.filesystem
-def test_data_context_create_raises_warning_and_leaves_existing_yml_untouched(
-    tmp_path_factory,
-):
-    project_path = str(tmp_path_factory.mktemp("data_context"))
-    FileDataContext.create(project_path)
-    ge_yml = os.path.join(project_path, "gx/great_expectations.yml")  # noqa: PTH118
-    with open(ge_yml, "a") as ff:
-        ff.write("# LOOK I WAS MODIFIED")
-
-    with pytest.warns(UserWarning):
-        FileDataContext.create(project_path)
-
-    with open(ge_yml) as ff:
-        obs = ff.read()
-    assert "# LOOK I WAS MODIFIED" in obs
-
-
-@pytest.mark.filesystem
-def test_data_context_create_makes_uncommitted_dirs_when_all_are_missing(
-    tmp_path_factory,
-):
-    project_path = str(tmp_path_factory.mktemp("data_context"))
-    FileDataContext.create(project_path)
```

Comment:
Do we want to remove these files, or just change them to call `gx.get_context(mode="file")`?

Reply:
Hmm no I think you're right - updated to keep most of them